index.html

```diff
@@ -1324,6 +1520,64 @@ <h2>DNS misconfiguration</h2>
 
       </section>
 
+      <section>
+        <h2>Distributed tracing</h2>
+
+        <pre class="example">
+&gt; GET / HTTP/1.1
+&gt; Host: example.com
+&gt; TraceParent: 00-0af7651916cd43dd8448eb211c80319c-b7ad6b7169203331-01
+
+&lt; HTTP/1.1 200 OK
+&lt; ...
+&lt; Report-To: {"group": "network-errors", "max_age": 2592000,
+              "endpoints": [{"url": "https://example.com/upload-reports"}]}
+&lt; NEL: {"report_to": "network-errors", "max_age": 2592000, "include_subdomains": true,
+        "request_headers": ["TraceParent"]}
+        </pre>
+
+        <p>
+        In this example, the owner of <code>example.com</code> makes requests to
+        several backend services when generating the response for an incoming
+        end user request, and uses a distributed tracing framework to correlate
+        events about all of these requests together.  The tracing framework
+        relies on the browser generating a unique trace ID for each outgoing
+        request, placing it in the request's <code>TraceParent</code> header.

```

Comment:
This bit assumes the browser supports such a thing, but that's not the case today, nor is it clear where / how that will be defined. Should we omit this for now? If we want to keep this, I'd propose pulling this content out into a non-normative note, with a link to relevant discussion / WIP spec that attempts to defines this.

Reply:
There was also confusion at TPAC about this — consensus was to discuss other use cases (as possibly remove/move this one as you suggest) to clarify that this feature is not specific to Distributed Tracing.

Reply:
👍

Reply:
Changed the example to talk about cache validation (`ETag` and `If-None-Match`) instead.